README.md

```diff
@@ -192,6 +192,40 @@ import * as stats from './stats';
 export { fallback } from './fallback';
 ```
 
+## How to add allowed word
+
+All the allowed words are located in an array in ` src/config.ts ` inside the object ` ALEX ` as shown below. 
```

Comment:
I think we can just link to the file - like this [`src/config.ts`](src/config.ts)

But, double check that the relative links work. 👍

Reply:
It works. I checked.